lib/iris/tests/unit/fileformats/grib/save_rules/test_shape_of_the_earth.py

```diff
@@ -110,6 +112,28 @@ def test_TransverseMercator_spherical(self):
         self._check_key('shapeOfTheEarth', 1)
         self._spherical_earth_test_common(radius)
 
+    def test__shape_of_earth_spherical(self):
+        cs = RotatedGeogCS(grid_north_pole_latitude=90.0,
+                           grid_north_pole_longitude=0.0,
+                           ellipsoid=GeogCS(52431.0))
+        test_cube = self._make_test_cube(cs=cs)
+        grid_definition_template_5(test_cube, self.mock_grib)
```

Comment:
As per my above. This test should return the correct results without needing to be run through this function first, I believe.